### ISSUE TITLE
Fails when right-clicking in the project view

### ISSUE THREAD

**Issue by munificent** [dart-atom/dartlang#219]
1. Have a directory of Dart code open in the project. I have core libraries open, not just a regular Dart package. Not sure if that's significant.
2. Right click a file or directory.

**Atom Version**: 1.0.7
**System**: Mac OS X 10.10.4
**Thrown From**: [dartlang](https://github.com/dart-atom/dartlang) package, v0.3.15

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/rnystrom/.atom/packages/dartlang/web/entry.dart.js:46108

TypeError: undefined is not a function
  at [object Object].J.getAttribute$1$x (/Users/rnystrom/.atom/packages/dartlang/web/entry.dart.js:46108:41)
  at AtomEvent.dart.AtomEvent.get$targetFilePath (/Users/rnystrom/.atom/packages/dartlang/web/entry.dart.js:20764:42)
  at PubContextCommand.dart.PubContextCommand.shouldDisplay$1 (/Users/rnystrom/.atom/packages/dartlang/web/entry.dart.js:24526:31)
  at ContextMenuItem_toJs_closure.dart.ContextMenuItem_toJs_closure.call$1 (/Users/rnystrom/.atom/packages/dartlang/web/entry.dart.js:20478:44)
  at [object Object].dart.Primitives_applyFunctionWithPositionalArguments (/Users/rnystrom/.atom/packages/dartlang/web/entry.dart.js:17458:28)
  at dart._callDartFunction (/Users/rnystrom/.atom/packages/dartlang/web/entry.dart.js:37998:31)
  at Object.shouldDisplay (/Users/rnystrom/.atom/packages/dartlang/web/entry.dart.js:37986:18)
  at ContextMenuManager.module.exports.ContextMenuManager.templateForEvent (/Users/rnystrom/Downloads/Atom.app/Contents/Resources/app.asar/src/context-menu-manager.js:108:25)
  at ContextMenuManager.module.exports.ContextMenuManager.showForEvent (/Users/rnystrom/Downloads/Atom.app/Contents/Resources/app.asar/src/context-menu-manager.js:166:27)
  at HTMLDocument.<anonymous> (/Users/rnystrom/Downloads/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:150:33)
  at HTMLDocument.handler (/Users/rnystrom/Downloads/Atom.app/Contents/Resources/app.asar/src/space-pen-extensions.js:112:34)
  at HTMLDocument.jQuery.event.dispatch (/Users/rnystrom/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/space-pen/vendor/jquery.js:4681:9)
  at HTMLDocument.elemData.handle (/Users/rnystrom/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/space-pen/vendor/jquery.js:4359:46)

```

### Commands

```
     -2:26.7.0 core:cancel (atom-text-editor.editor.is-focused)
     -1:17 application:add-project-folder (atom-text-editor.editor.is-focused)
  3x -0:52 linter:next-error (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
     -0:42.1.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:42.1.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:40.2.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:40.2.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:38.9.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:38.9.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:35.9.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:35.9.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:33.8.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:33.8.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:33.5.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:33.5.0 core:cancel (atom-text-editor.editor.is-focused)
  5x -0:32.8.0 linter:next-error (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "tabs",
      "git-diff"
    ]
  },
  "dartlang": {
    "sdkLocation": "/Users/rnystrom/dev/dart/sdk/xcodebuild/ReleaseIA32/dart-sdk"
  }
}
```

### Installed Packages

```coffee
# User
dartlang, v0.3.15
linter, v1.3.4

# Dev
No dev packages
```

**Comment by devoncarew** [comment]
The good news is you're on the latest atom and dartlang plugin versions. The bad news is that the old dartlang plugin had this exception, and the latest one shouldn't. I don't see the issue when running locally. I'll investigate a bit more.

**Comment by devoncarew** [comment]
fixed in `0.3.16`
